File: Assignment 2/Q14.c
#include <stdio.h>

int main()
{
    int x, y;
    printf("ENter the value of X:");
    scanf("%d", &x);
    switch (x)
    {
    case 6:
        y = x + 10;
        break;
    case 7:
        y = x * x;
        break;
    case 12:
        y = 2 * x + 4;
        break;
    default:
        y = x * (6 - 1);
    }
    printf("The value of Y is: %d", y);
    return 0;
}

File: Assignment 2/Q14.py
x = int(input("Enter the value of X: "))

if x == 6:
    y = x + 10
elif x == 7:
    y = x * x
elif x == 12:
    y = 2 * x + 4
else:
    y = x * (6 - 1)

print(f"The value of Y is: {y}")